README.md

```diff
@@ -3,3 +3,24 @@
 ![Build Status](https://flowcrypt.semaphoreci.com/badges/flowcrypt-ios.svg?key=9bd38bf4-4a38-4cb3-b551-38302af1eb07)
 
 FlowCrypt iOS App, download from https://flowcrypt.com/download
+
+### Installation
+
+You will need to have Xcode *(version 12.4)* installed
+* clone the repo
+```sh
+git clone https://github.com/FlowCrypt/flowcrypt-ios.git
+cd flowcrypt-ios
+```
+* install project dependencies
+```sh
+bundle install --path vendor/bundle
+```
+* install project Pods
+``` sh
+bundle exec pod install
```

Comment:
I get an error at this line of the instructions. Here's my entire session:

```
alexalvarado@Alexs-MacBook-Air Downloads % git clone git@github.com:FlowCrypt/flowcrypt-ios.git
Cloning into 'flowcrypt-ios'...
Warning: Permanently added the RSA host key for IP address '140.82.114.4' to the list of known hosts.
remote: Enumerating objects: 199, done.
remote: Counting objects: 100% (199/199), done.
remote: Compressing objects: 100% (136/136), done.
remote: Total 3809 (delta 77), reused 136 (delta 59), pack-reused 3610
Receiving objects: 100% (3809/3809), 4.41 MiB | 1.24 MiB/s, done.
Resolving deltas: 100% (2180/2180), done.
alexalvarado@Alexs-MacBook-Air Downloads % cd flowcrypt-ios 
alexalvarado@Alexs-MacBook-Air flowcrypt-ios % git checkout feature/update-readme
Branch 'feature/update-readme' set up to track remote branch 'feature/update-readme' from 'origin'.
Switched to a new branch 'feature/update-readme'
alexalvarado@Alexs-MacBook-Air flowcrypt-ios % bundle install --path vendor/bundle

Warning: the running version of Bundler (1.17.2) is older than the version that created the lockfile (1.17.3). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Fetching gem metadata from https://rubygems.org/........
Fetching rake 13.0.3
Installing rake 13.0.3
Fetching CFPropertyList 3.0.3
Installing CFPropertyList 3.0.3
Fetching concurrent-ruby 1.1.8
Installing concurrent-ruby 1.1.8
Fetching i18n 1.8.9
Installing i18n 1.8.9
Fetching minitest 5.14.4
Installing minitest 5.14.4
Fetching thread_safe 0.3.6
Installing thread_safe 0.3.6
Fetching tzinfo 1.2.9
Installing tzinfo 1.2.9
Fetching activesupport 5.2.4.5
Installing activesupport 5.2.4.5
Fetching public_suffix 4.0.6
Installing public_suffix 4.0.6
Fetching addressable 2.7.0
Installing addressable 2.7.0
Fetching httpclient 2.8.3
Installing httpclient 2.8.3
Fetching json 2.5.1
Installing json 2.5.1 with native extensions
Fetching algoliasearch 1.27.5
Installing algoliasearch 1.27.5
Fetching artifactory 3.0.15
Installing artifactory 3.0.15
Fetching atomos 0.1.3
Installing atomos 0.1.3
Fetching aws-eventstream 1.1.1
Installing aws-eventstream 1.1.1
Fetching aws-partitions 1.434.0
Installing aws-partitions 1.434.0
Fetching aws-sigv4 1.2.3
Installing aws-sigv4 1.2.3
Fetching jmespath 1.4.0
Installing jmespath 1.4.0
Fetching aws-sdk-core 3.113.0
Installing aws-sdk-core 3.113.0
Fetching aws-sdk-kms 1.43.0
Installing aws-sdk-kms 1.43.0
Fetching aws-sdk-s3 1.92.0
Installing aws-sdk-s3 1.92.0
Fetching babosa 1.0.4
Installing babosa 1.0.4
Using bundler 1.17.2
Fetching claide 1.0.3
Installing claide 1.0.3
Fetching fuzzy_match 2.0.4
Installing fuzzy_match 2.0.4
Fetching nap 1.1.0
Installing nap 1.1.0
Fetching netrc 0.11.0
Installing netrc 0.11.0
Fetching ffi 1.15.0
Installing ffi 1.15.0 with native extensions
Fetching ethon 0.12.0
Installing ethon 0.12.0
Fetching typhoeus 1.4.0
Installing typhoeus 1.4.0
Fetching cocoapods-core 1.10.1
Installing cocoapods-core 1.10.1
Fetching cocoapods-deintegrate 1.0.4
Installing cocoapods-deintegrate 1.0.4
Fetching cocoapods-downloader 1.4.0
Installing cocoapods-downloader 1.4.0
Fetching cocoapods-plugins 1.0.0
Installing cocoapods-plugins 1.0.0
Fetching cocoapods-search 1.0.0
Installing cocoapods-search 1.0.0
Fetching cocoapods-trunk 1.5.0
Installing cocoapods-trunk 1.5.0
Fetching cocoapods-try 1.2.0
Installing cocoapods-try 1.2.0
Fetching colored2 3.1.2
Installing colored2 3.1.2
Fetching escape 0.0.4
Installing escape 0.0.4
Fetching fourflusher 2.3.1
Installing fourflusher 2.3.1
Fetching gh_inspector 1.1.3
Installing gh_inspector 1.1.3
Fetching molinillo 0.6.6
Installing molinillo 0.6.6
Fetching ruby-macho 1.4.0
Installing ruby-macho 1.4.0
Fetching nanaimo 0.3.0
Installing nanaimo 0.3.0
Fetching xcodeproj 1.19.0
Installing xcodeproj 1.19.0
Fetching cocoapods 1.10.1
Installing cocoapods 1.10.1
Fetching colored 1.2
Installing colored 1.2
Fetching highline 1.7.10
Installing highline 1.7.10
Fetching commander-fastlane 4.4.6
Installing commander-fastlane 4.4.6
Fetching declarative 0.0.20
Installing declarative 0.0.20
Fetching declarative-option 0.1.0
Installing declarative-option 0.1.0
Fetching digest-crc 0.6.3
Installing digest-crc 0.6.3 with native extensions
Fetching unf_ext 0.0.7.7
Installing unf_ext 0.0.7.7 with native extensions
Fetching unf 0.1.4
Installing unf 0.1.4
Fetching domain_name 0.5.20190701
Installing domain_name 0.5.20190701
Fetching dotenv 2.7.6
Installing dotenv 2.7.6
Fetching emoji_regex 3.2.2
Installing emoji_regex 3.2.2
Fetching excon 0.79.0
Installing excon 0.79.0
Fetching faraday-net_http 1.0.1
Installing faraday-net_http 1.0.1
Fetching multipart-post 2.0.0
Installing multipart-post 2.0.0
Fetching ruby2_keywords 0.0.4
Installing ruby2_keywords 0.0.4
Fetching faraday 1.3.0
Installing faraday 1.3.0
Fetching http-cookie 1.0.3
Installing http-cookie 1.0.3
Fetching faraday-cookie_jar 0.0.7
Installing faraday-cookie_jar 0.0.7
Fetching faraday_middleware 1.0.0
Installing faraday_middleware 1.0.0
Fetching fastimage 2.2.3
Installing fastimage 2.2.3
Fetching jwt 2.2.2
Installing jwt 2.2.2
Fetching memoist 0.16.2
Installing memoist 0.16.2
Fetching multi_json 1.15.0
Installing multi_json 1.15.0
Fetching os 1.1.1
Installing os 1.1.1
Fetching signet 0.15.0
Installing signet 0.15.0
Fetching googleauth 0.16.0
Installing googleauth 0.16.0
Fetching mini_mime 1.0.2
Installing mini_mime 1.0.2
Fetching uber 0.1.0
Installing uber 0.1.0
Fetching representable 3.0.4
Installing representable 3.0.4
Fetching retriable 3.1.2
Installing retriable 3.1.2
Fetching google-api-client 0.38.0
Installing google-api-client 0.38.0
Fetching rexml 3.2.4
Installing rexml 3.2.4
Fetching webrick 1.7.0
Installing webrick 1.7.0
Fetching google-apis-core 0.3.0
Installing google-apis-core 0.3.0
Fetching google-apis-iamcredentials_v1 0.2.0
Installing google-apis-iamcredentials_v1 0.2.0
Fetching google-apis-storage_v1 0.3.0
Installing google-apis-storage_v1 0.3.0
Fetching google-cloud-env 1.5.0
Installing google-cloud-env 1.5.0
Fetching google-cloud-errors 1.1.0
Installing google-cloud-errors 1.1.0
Fetching google-cloud-core 1.6.0
Installing google-cloud-core 1.6.0
Fetching google-cloud-storage 1.31.0
Installing google-cloud-storage 1.31.0
Fetching mini_magick 4.11.0
Installing mini_magick 4.11.0
Fetching naturally 2.2.1
Installing naturally 2.2.1
Fetching plist 3.6.0
Installing plist 3.6.0
Fetching rubyzip 2.3.0
Installing rubyzip 2.3.0
Fetching security 0.1.3
Installing security 0.1.3
Fetching simctl 1.6.8
Installing simctl 1.6.8
Fetching slack-notifier 2.3.2
Installing slack-notifier 2.3.2
Fetching terminal-notifier 2.0.0
Installing terminal-notifier 2.0.0
Fetching unicode-display_width 1.7.0
Installing unicode-display_width 1.7.0
Fetching terminal-table 1.8.0
Installing terminal-table 1.8.0
Fetching tty-screen 0.8.1
Installing tty-screen 0.8.1
Fetching tty-cursor 0.7.1
Installing tty-cursor 0.7.1
Fetching tty-spinner 0.9.3
Installing tty-spinner 0.9.3
Fetching word_wrap 1.0.0
Installing word_wrap 1.0.0
Fetching rouge 2.0.7
Installing rouge 2.0.7
Fetching xcpretty 0.3.0
Installing xcpretty 0.3.0
Fetching xcpretty-travis-formatter 1.0.1
Installing xcpretty-travis-formatter 1.0.1
Fetching fastlane 2.178.0
Installing fastlane 2.178.0
Fetching fastlane-plugin-semaphore 0.2.0
Installing fastlane-plugin-semaphore 0.2.0
Fetching http-accept 1.7.0
Installing http-accept 1.7.0
Fetching mime-types-data 3.2021.0225
Installing mime-types-data 3.2021.0225
Fetching mime-types 3.3.1
Installing mime-types 3.3.1
Fetching rest-client 2.1.0
Installing rest-client 2.1.0
Fetching xcode-install 2.6.8
Installing xcode-install 2.6.8
Bundle complete! 5 Gemfile dependencies, 111 gems now installed.
Bundled gems are installed into `./vendor/bundle`
Post-install message from algoliasearch:
A new major version is available for Algolia! Please now use the https://rubygems.org/gems/algolia gem to get the latest features.
alexalvarado@Alexs-MacBook-Air flowcrypt-ios % bundle exec pod install

Analyzing dependencies

――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

### Command

```
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/bin/pod install
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?


### Stack

```
   CocoaPods : 1.10.1
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.arm64e-darwin20]
    RubyGems : 3.0.3
        Host : macOS 11.0.1 (20B29)
       Xcode : 12.3 (12C33)
         Git : git version 2.24.3 (Apple Git-128)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib
Repositories : 
```

### Plugins

```
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-trunk       : 1.5.0
cocoapods-try         : 1.2.0
```

### Podfile

```ruby
# Uncomment the next line to define a global platform for your project
platform :ios, '12.2'

use_frameworks!

############################ Pods ############################

def app_pods
  pod 'GoogleSignIn'
  pod 'mailcore2-ios'
  pod 'MBProgressHUD'
  pod 'SwiftLint' # todo - add linting rules
  pod 'Toast', '~> 4.0.0'
  pod 'ENSwiftSideMenu', '~> 0.1.4'
  pod 'Texture'
  pod 'SwiftLint'
  pod 'SwiftFormat/CLI'
end

def shared_pods
  pod 'RealmSwift'
  pod 'PromisesSwift'
  pod 'SwiftyRSA'
  pod 'IDZSwiftCommonCrypto'
  pod 'mailcore2-ios'
  pod 'BigInt', '~> 5.0'
end

def ui_pods
  pod 'Texture'
end

def google_pods
  pod 'GoogleAPIClientForREST/Gmail'
end

############################ Targets ############################
target 'FlowCrypt' do
  shared_pods
  app_pods
  google_pods
end

target 'FlowCryptTests' do
  shared_pods
  inherit! :search_paths
end

target 'FlowCryptUI' do
  ui_pods
end

target 'FlowCryptUIApplication' do
  ui_pods
end

target 'FlowCryptUITests' do
  pod 'GoogleSignIn'
end


## Set IPHONEOS_DEPLOYMENT_TARGET for all pods
post_install do |pi|
    pi.pods_project.targets.each do |t|
      t.build_configurations.each do |config|
        config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.0'
      end
    end
end
```

### Error

```
LoadError - dlopen(/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ffi-1.15.0/lib/ffi_c.bundle, 0x0009): missing compatible arch in /Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ffi-1.15.0/lib/ffi_c.bundle - /Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ffi-1.15.0/lib/ffi_c.bundle
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ffi-1.15.0/lib/ffi.rb:6:in `require'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ffi-1.15.0/lib/ffi.rb:6:in `rescue in <top (required)>'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ffi-1.15.0/lib/ffi.rb:3:in `<top (required)>'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ethon-0.12.0/lib/ethon.rb:2:in `require'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/ethon-0.12.0/lib/ethon.rb:2:in `<top (required)>'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/typhoeus-1.4.0/lib/typhoeus.rb:2:in `require'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/typhoeus-1.4.0/lib/typhoeus.rb:2:in `<top (required)>'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/sources_manager.rb:74:in `require'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/sources_manager.rb:74:in `cdn_url?'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/sources_manager.rb:36:in `create_source_with_url'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/sources_manager.rb:21:in `find_or_create_source_with_url'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:178:in `block in sources'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:177:in `map'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:177:in `sources'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:1073:in `block in resolve_dependencies'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:1072:in `resolve_dependencies'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:124:in `analyze'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:414:in `analyze'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:239:in `block in resolve_dependencies'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:238:in `resolve_dependencies'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:160:in `install!'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/command/install.rb:52:in `run'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/lib/cocoapods/command.rb:52:in `run'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/gems/cocoapods-1.10.1/bin/pod:55:in `<top (required)>'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/bin/pod:23:in `load'
/Users/alexalvarado/Downloads/flowcrypt-ios/vendor/bundle/ruby/2.6.0/bin/pod:23:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `load'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `kernel_load'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli/exec.rb:28:in `run'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli.rb:463:in `exec'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli.rb:27:in `dispatch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli.rb:18:in `start'
/Library/Ruby/Gems/2.6.0/gems/bundler-1.17.2/exe/bundle:30:in `block in <top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/friendly_errors.rb:124:in `with_friendly_errors'
/Library/Ruby/Gems/2.6.0/gems/bundler-1.17.2/exe/bundle:22:in `<top (required)>'
/usr/bin/bundle:23:in `load'
/usr/bin/bundle:23:in `<main>'

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.

Search for existing GitHub issues similar to yours:
https://github.com/CocoaPods/CocoaPods/search?q=dlopen%28%2FUsers%2Falexalvarado%2FDownloads%2Fflowcrypt-ios%2Fvendor%2Fbundle%2Fruby%2F2.6.0%2Fgems%2Fffi-1.15.0%2Flib%2Fffi_c.bundle%2C+0x0009%29%3A+missing+compatible+arch+in+%2FUsers%2Falexalvarado%2FDownloads%2Fflowcrypt-ios%2Fvendor%2Fbundle%2Fruby%2F2.6.0%2Fgems%2Fffi-1.15.0%2Flib%2Fffi_c.bundle+-+%2FUsers%2Falexalvarado%2FDownloads%2Fflowcrypt-ios%2Fvendor%2Fbundle%2Fruby%2F2.6.0%2Fgems%2Fffi-1.15.0%2Flib%2Fffi_c.bundle&type=Issues

If none exists, create a ticket, with the template displayed above, on:
https://github.com/CocoaPods/CocoaPods/issues/new

Be sure to first read the contributing guide for details on how to properly submit a ticket:
https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md

Don't forget to anonymize any private data!

Looking for related issues on cocoapods/cocoapods...
Searching for inspections failed: undefined method `map' for nil:NilClass
alexalvarado@Alexs-MacBook-Air flowcrypt-ios % 
```

Reply:
I can get a little further by putting `arch -x86_64` in front of all of the commands (so they run through Rosetta). Using this approach, I get errors when trying to run the project in Xcode:

1. ld: in /Users/alexalvarado/Downloads/flowcrypt-ios/Pods/Realm/core/librealmcore-ios.a(bptree.o), building for iOS Simulator, but linking in object file built for iOS, for architecture arm64
2. No profiles for 'com.flowcrypt.FlowCryptUITests.xctrunner' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.flowcrypt.FlowCryptUITests.xctrunner'.
3. Command CompileSwift failed with a nonzero exit code
4. There are no accounts registered with Xcode. Add your developer account to Xcode (this one at least seems pretty straightforward)

Reply:
I can get a little further by prepending each command with `arch -x86_64` to run it through Rosetta. But then when I try to run the project in Xcode I get four errors:

1. There are no accounts registered with Xcode. Add your developer account to Xcode (this one seems pretty straightforward at least)
2. No profiles for 'com.flowcrypt.FlowCryptUITests.xctrunner' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'com.flowcrypt.FlowCryptUITests.xctrunner'.
3. ld: in /Users/alexalvarado/Downloads/flowcrypt-ios/Pods/Realm/core/librealmcore-ios.a(bptree.o), building for iOS Simulator, but linking in object file built for iOS, for architecture arm64
4. Command CompileSwift failed with a nonzero exit code

Reply:
Are you running it on M1 Mac?

Reply:
Yes, I'm using Xcode 12.3 on MacOS Big Sur 11.0.1 on an M1 chip

Reply:
Not sure app supports running on M1. 
Based on your logs it looks like Realm doesn't support it.
We need to update Realm (hope it's only one library which doesn't support arm64)

Let me try to update Realm,  or I will need to exclude arm64 from the simulator architectures in the app. 
Will keep you posted